{
    "type": "types.OneOf",
    "description": "Succeeded auth attempt",
    "data": {
        "AuthAttempt": {
            "Metadata": {
                "Index": "1",
                "Type": "auth",
                "ID": "{{ uuid }}",
                "Code": "{{ uuid }}",
                "Time": "{{ time }}",
                "ClusterName": "test-cluster"
            },
            "User": {
                "User": "foo",
                "Login": "foo",
                "Impersonator": "foo",
                "AWSRoleARN": "arn:aws:iam::account:root",
                "AccessRequests": [
                    "{{ uuid }}"
                ]
            },
            "Connection": {
                "LocalAddr": "127.0.0.1",
                "RemoteAddr": "192.168.0.1",
                "Protocol": "https"
            },
            "Status": {
                "Success": false,
                "Error": "An error occured",
                "UserMessage": "Access denied"
            }
        }
    }
}